lua/nvim-tree/live-filter.lua

```diff
@@ -2,6 +2,9 @@ local view = require "nvim-tree.view"
 local utils = require "nvim-tree.utils"
 local Iterator = require "nvim-tree.iterators.node-iterator"
 
+-- Keeps memory of focused node when entering filter prompt
+local last_node
```

Comment:
This will be a future problem when we finally sort out multiple trees / tabs.

Can you please store this at `view.View`?